app/policies/namespace_policy.rb

```diff
@@ -99,11 +99,7 @@ def initialize(user, scope)
 
     def resolve
       if user.admin?
-        global = scope.where(global: true)
-        normal = scope.not_portus
-                      .where(global: false)
-                      .order(created_at: :asc)
-        global + normal
+        scope.not_portus.order(created_at: :asc)
```

Comment:
Seriously? :man_facepalming:

Reply:
Yeah... Not sure what I was using when I wrote that code. Haha.